src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/entities/RemissionType.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.iacasedocumentsapi.domain.entities;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum RemissionType {
```

Comment:
Can we please have the RemissionTypeTest.java from ia-case-api too? So that the coverage doesn't drop in SonarQube

Reply:
Thank for the catch, added this test